ibis/expr/operations.py

```diff
@@ -1025,6 +1025,15 @@ def __init__(self, expr, window):
         if table is not None:
             window = window.bind(table)
 
+        if window.max_lookback is not None:
+            error_msg = ("'max lookback windows must be ordered "
```

Comment:
Is this missing a single quote after `lookback`?

Reply:
Yes, sadly...not anymore though :)